SPECS/python-pycryptodome/python-pycryptodome.spec

```diff
@@ -0,0 +1,419 @@
+#
+# spec file for package python-pycryptodome
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+%{?!python_module:%define python_module() python-%{**} python3-%{**}}
+%define oldpython python
+Name:           python-pycryptodome
+Version:        3.14.1
+Release:        3
+Summary:        Cryptographic library for Python
+License:        BSD-2-Clause
```

Comment:
We should use the [Fedora short name](https://docs.fedoraproject.org/en-US/packaging-guidelines/LicensingGuidelines/#_valid_license_short_names) for our license tags for consistency. In this case, it should be `BSD`.

Reply:
Understood.